### ISSUE TITLE
Floating windows doesn't follow when you change workspaces

### ISSUE THREAD

**Issue by JustSimplyKyle** [hyprwm/Hyprland#1129]
https://user-images.githubusercontent.com/68589851/204570478-580b0e73-820f-49e9-ba22-a007dc6f43df.mp4

Basically the floating window disappears when you are switching workspaces.(`hyprctl dispatch workspace x`)

**Comment by Dickby** [comment]
There is a dispatcher `pin` that pins the active floating window.

**Comment by JustSimplyKyle** [comment]
No, I don't want to pin it. I just want to move the window to another workspace while dragging it

**Comment by vaxerski** [comment]
dupe of #988 then
